src/Packagist/WebBundle/Resources/config/services.yml

```diff
@@ -26,14 +26,12 @@ services:
 
     packagist.user_repository:
         class: Packagist\WebBundle\Entity\UserRepository
-        factory_service: doctrine
-        factory_method: getRepository
+        factory: [@doctrine, getRepository]
```

Comment:
Might as well go ahead and quote all `@<something>` references (in preparation for SF 2.8/3.x).